### ISSUE TITLE
New Package: Numeronyms

### ISSUE THREAD

**Issue by omrilotan** [federico-terzi/espanso-hub#47]
https://github.com/omrilotan/espanso-package-numeronyms

Espanso package containing commonly used numeronyms.

![](https://user-images.githubusercontent.com/516342/103480613-efba3080-4ddd-11eb-869b-38545eb00b59.gif)

A numeronym is a word where a number is used to form an abbreviation.

**Comment by kopach** [comment]
looks really good

**Comment by federico-terzi** [comment]
Hey!

First of all, sorry for the late reply! The new v2 took a long time to complete, so I had to temporarily focus my efforts there :)
Thank you for the help with the package! I've added it to the new [hub repository](https://github.com/espanso/hub), which will be the new standard location for "verified" packages.

A couple of notes:
* This new hub is only accessible through Espanso v2, so if you are using the legacy Espanso version (0.7.3), you 
won't be able to install this package yet. 
After upgrading, you'll be able to install the package with the usual `espanso install <package_name>`
* The hub at [hub.espanso.org](https://hub.espanso.org) is still showing the old repository with the legacy format, so this package is not visible yet.
We are working hard to create a new (and improved!) Hub. Once that will be ready, your package will automatically show there :)
* The v2 uses a slightly different package format, so I converted the package before adding it to the hub.
If you still want to support package installations from the external repo, we might need to commit a couple of updates
to your repo.

The v2 package documentation is not yet there, but I'm working hard to get it up to speed soon :)

Again, thank you for the help, and happy expanding! Feel free to let me know of any concerns you might have

Cheers :)
